modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -694,55 +731,57 @@ private[codegen] class Renderer(compilationUnit: CompilationUnit) { self =>
   }
 
   implicit class TypeExt(tpe: Type) {
+    val schemaPkg_ = "smithy4s.schema.Schema"
     def schemaRef: Line = tpe match {
-      case Type.PrimitiveType(p) => Line(schemaRefP(p))
+      case Type.PrimitiveType(p) => TypeReference(schemaRefP(p))
```

Comment:
let's change the `TypeReference` type to `NameRef` or something, because : 

1. it's shorter 
2. we're not just referencing types